docs/volume_annotation/tools.md

```diff
@@ -1,64 +1,128 @@
 
-![youtube-video](https://www.youtube.com/embed/JkpSTKuNZKg)
+![Change Brush Size Modifier](./images/brush-size-modifier.jpg){align=left width="60"} 
+**Change Brush Size**: Changes the size and radius of the brushing tool. Presets of small, medium and large brush sizes are available which can be modified to suite your workflows. ![youtube-video](https://www.youtube.com/embed/JkpSTKuNZKg)
 
-In the `Segmentation` tab on the right-hand side panel, you can see the segment IDs which are available in your annotation. You can rename segments as needed.
+![Overwrite Everything Modifier](./images/overwrite-everything-modifier.jpg){align=left width="60"} 
+**Overwrite Everything**: When labeling with the brush/trace tool, you can annotate every available voxel without any regard if it was already labeled as a different segment or whether it is unlabelled. This allows you to draw over existing segments.
+
+![Overwrite Empty Voxels Modifier](./images/overwrite-empty-modifier.jpg){align=left width="60"} 
+**Only Overwrite Empty Areas**: In contrast to the `Overwrite Everything` modifier, the forces the brush & trace tools to only label voxels without any segment ID ("empty areas"). This is useful when annotating segments that directly touch each other to avoid accidental overwrites.
+
+![Interpolation/Extrusion Modifier](./images/interpolation-modifier.jpg){align=left width="60"} 
+**Interpolation/Extrusion**: Annotate a segment, skip a few sections in the Z direction, and annotate it again. Now, you can interpolate between the two segments. Read more on the [interpolation/extrusion](#volume-interpolation) below. 
+
+![2D Fill Modifier](./images/2d-modifier.jpg){align=left width="60"} 
+![3D Fill Modifier](./images/3d-modifier.jpg){align=left width="60"} 
+**2D/3D Fill**: Modifies the flood filling tool to work in 2D (in-plane only) or 3D (volumetric fill/re-labeling). 3D flood fill is constrained to a small, regional bounding box for performance reasons. Read more about [flood fills](#volume-flood-fills) below.
 
-The active segment ID under the cursor can be found in the status bar at the bottom of the screen or through the context-sensitive menu on right-click.
 
 ## Quick-select tool
-The built-in quick select tools allows you draw a selection around a cell or object and WEBKNOSSOS will use machine-learning to automatically do the segmentation for you. The feature is based on the [Segment Anything Model 2](https://ai.meta.com/blog/segment-anything-2/) and works across a wide range of imaging modalities.
+The Quick Select tool offers AI-powered automatic segmentation, powered by [Segment Anything Model 2](https://ai.meta.com/blog/segment-anything-2/). Simply draw a selection around your target structure, and WEBKNOSSOS will automatically segment it for you.
+
+### Operating Modes
+
+**AI Mode** (Default)
+
+- Activate the "AI" button in the toolbar
```

Comment:
probably nitpicky, but it's a bit weird that the headline says that it's enabled by default and then the description contains the imperative "activate it".
maybe move the part about activating/deactivating directly into the intro of "Operating Modes".